### PR TITLE
[Doc] Documenting a subset of the entropy 0.4 format

### PR DESCRIPTION
**Pull request by Yoric**
This documentation/spec is very incomplete as of yet, but I'm planning to improve it gradually.

### CODE REVIEW

File: spec/entropy.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+a `CompressedByteStream<T>` with the specified `CompressionFormat`.
+
+## Stream interpretation
+
+### Integers
+
+```
+ExpandedByteStream<u32?> ::= ExpandedByteStreamItem<u32?>*
+ExpandedByteStreamItem<u32?> ::= Varu32         // Some(value)
+                              | Varu32Invalid1  // None
```
Comment (arai-a) [inline spec/entropy.md]: one more space for indent

File: spec/entropy.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+```
+
+TBD
+
+### Floating-point
+
+```
+ExpandedByteStream<f64?> ::= ExpandedByteStreamItem<f64?>
+ExpandedByteStreamItem<f64?> ::= Vari32            // Some(integer value)
+                              | Varu32Invalid1 f64 // Some(floating-point value)
```
Comment (arai-a) [inline spec/entropy.md]: here too

File: spec/entropy.md (hunk 1 above)
Comment (Yoric) [reply spec/entropy.md]: Gasp, I learnt to center the `|` under the `::=` :)

**Comment by Yoric** [comment]
I took the opportunity to add some more doc, so you may want to take a second look before I merge.

**Pull request merged**